src/test/scala/io/iohk/ethereum/vm/OpCodeGasSpec.scala

```diff
@@ -477,19 +477,19 @@ class OpCodeGasSpec extends FunSuite with OpCodeTesting with Matchers with Prope
     // Sending refund to a non-existent account
     forAll(stateGen) { stateIn =>
       val (refund, _) = stateIn.stack.pop
-      whenever(stateIn.world.getAccount(Address(refund)).isEmpty) {
+      whenever(stateIn.world.getAccount(Address(refund)).isEmpty && stateIn.ownBalance > 0) {
         val stateOut = op.execute(stateIn)
         stateOut.gasRefund shouldEqual R_selfdestruct
         verifyGas(G_selfdestruct + G_newaccount, stateIn, stateOut)
       }
     }
 
-    // Sending refund to an already existing account
+    // Sending refund to an already existing account not dead account
     forAll(stateGen) { (stateIn) =>
       val (refund, _) = stateIn.stack.pop
       val world = stateIn.world.saveAccount(
         Address(refund),
-        Account.empty())
+        Account.empty().increaseNonce())
```

Comment:
If we were to leave that EVM config - why is this needed?

Reply:
It is only needed because of EIP161 config, as an empty account is considered dead and there are different fees.